### ISSUE TITLE
Minor bugs on Debian 10

### ISSUE THREAD

**Issue by Zapunidi** [saveriomiroddi/zfs-installer#78]
I ran install script under Debian 10 live. Everything worked OK eventually, but there were some problems.
1) After first reboot the root filesystem do not mount by initramfs because zfs was mounted by other system and therefore requires force import. Running zpool import -f rpool; under initramfs prompt fails because / is already mounted. Rebooting once again helps. I understand it happened because script failed somewhere close to the end. There were some errors about rsync not found. I will run it once again and provide more precise information.
2) Installer manual says: - set `Storage device` to `Unknown - 10.0 GB '"${v_temp_volume_device}"'` while in my case the partition size was 16G, not 10G.

**Comment by Zapunidi** [comment]
1. Install script fails after trying to sync_efi _partitions because rsync is not found in chroot.
```
#############################################################################
# sync_efi_partitions
###############################################################################
bash: rsync: command not found
```
2. Found solution. Just need to update help. I will do it myself and send the pull request.

I have attached zfs-installer logs
[disks.log](https://github.com/saveriomiroddi/zfs-installer/files/4496433/disks.log)
[install.log](https://github.com/saveriomiroddi/zfs-installer/files/4496434/install.log)
[lsb_release.log](https://github.com/saveriomiroddi/zfs-installer/files/4496435/lsb_release.log)
[updated_module_versions.log](https://github.com/saveriomiroddi/zfs-installer/files/4496436/updated_module_versions.log)

**Comment by Zapunidi** [comment]
I have created pull request with all fixes, Please accept.

**Comment by saveriomiroddi** [comment]
> Installer manual says: - set Storage device to Unknown - 10.0 GB '"${v_temp_volume_device}"' while in my case the partition size was 16G, not 10G.

Nice catch :slightly_smiling_face: I've fixed this and released. Note that the new size displayed is an approximation (the format is not exactly the same), but it shall do :slightly_smiling_face:.

**Comment by Zapunidi** [comment]
I can't test it again due to lack of free hardware. So no feedback here, sorry.

**Comment by saveriomiroddi** [comment]
Closed by #103.
